src/OpenTelemetry/Metrics/Exemplar/ExemplarFilterType.cs

```diff
@@ -0,0 +1,61 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+using System.Diagnostics;
+#if EXPOSE_EXPERIMENTAL_FEATURES && NET8_0_OR_GREATER
+using System.Diagnostics.CodeAnalysis;
+using OpenTelemetry.Internal;
+#endif
+
+namespace OpenTelemetry.Metrics;
+
+#if EXPOSE_EXPERIMENTAL_FEATURES
+/// <summary>
+/// Defines the exemplar filters supported by the OpenTelemetry .NET Metrics
+/// SDK.
+/// </summary>
+/// <remarks>
+/// <inheritdoc cref="Exemplar" path="/remarks/para[@experimental-warning='true']"/>
+/// Specification: <see
+/// href="https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk.md#exemplarfilter"/>.
+/// </remarks>
+#if NET8_0_OR_GREATER
+[Experimental(DiagnosticDefinitions.ExemplarExperimentalApi, UrlFormat = DiagnosticDefinitions.ExperimentalApiUrlFormat)]
+#endif
+public
+#else
+internal
+#endif
+    enum ExemplarFilterType
+{
+    /// <summary>
+    /// An exemplar filter which makes no measurements eligible for becoming an
+    /// <see cref="Exemplar"/>.
```

Comment:
```suggestion
    /// <see cref="Exemplar"/>. This is the equivalent of disabling Exemplars feature.
```

should we tell something like this?

Reply:
Added a note in remarks: https://github.com/open-telemetry/opentelemetry-dotnet/pull/5404/commits/0884011a546a1121885848ce5db97317accfb2a1